imgui.h

```diff
@@ -23,6 +23,13 @@
 #define IMGUI_API
 #endif
 
+// Calling conventions
+#if !defined(IMGUI_CALLBACK) && defined(_WIN32)
+#define IMGUI_CALLBACK __cdecl
+#else
+#define IMGUI_CALLBACK
+#endif
+
```

Comment:
Yes but isn't the point that the user might want to define IMGUI_CALLBACK to something that's not your default?

Shouldn't the correct test should just be:

```
#ifndef IMGUI_CALLBACK
#define IMGUI_CALLBACK
#endif
```

Let the user do what they want and not involve _WIN32 there?